src/utils.ts

```diff
@@ -252,3 +252,20 @@ export function spanLogic(
     rightLastVisible,
   };
 }
+
+export function getMaxDepth(rootNodes:Array<any>){
+  let levels:number[]=[];
+  const addToArray = (node: ILabelNode)=>{
+    levels.push(node.level)
+    if (node.children.length === 0) {
```

Comment:
isn't it just `return node.expand && node.children.length > 0`

Reply:
done